app/src/main/res/layout/studies_section_item.xml

```diff
@@ -24,7 +24,6 @@
         android:layout_width="match_parent"
         android:layout_height="@dimen/section_header_height"
         android:gravity="start|center_vertical"
-        app:fontFamily="@font/metropolis_semibold"
```

Comment:
Same as above.

Reply:
Same as others: this already had bold style applied, and it seemed to match better with the layout. So I just removed the font here.